app/src/components/market/common/position_table/index.tsx

```diff
@@ -58,8 +59,12 @@ export const PositionTable = (props: Props) => {
 
   const shortShares = balances[0].shares
   const longShares = balances[1].shares
-  const shortSharesFormatted = formatNumber(formatBigNumber(shortShares || new BigNumber(0), collateral.decimals))
-  const longSharesFormatted = formatNumber(formatBigNumber(longShares || new BigNumber(0), collateral.decimals))
+  const shortSharesFormatted = formatNumber(
+    formatBigNumber(shortShares || new BigNumber(0), collateral.decimals, STANDARD_DECIMALS),
```

Comment:
Have to make sure all the `formatBigNumber` instances are not using `STANDARD_DECIMALS` as the third param if the second param is a variable value like collateral.decimals

Reply:
got it now, thanks, appreciate your patience w me